.replit

```diff
@@ -0,0 +1,2 @@
+language = "nodejs"
+run = "npm install && cd example && node array.js"
```

Comment:
```suggestion
run = "npm install && cd example && node array"
```
altho i wonder if we should make an `index.js` that requires all the other files, so we can run all the examples here?